modules/sdk-core/src/bitgo/lightning/lightning.ts

```diff
@@ -35,7 +36,10 @@ export class Lightning implements ILightning {
   }
 
   public async getBalance(): Promise<GetBalanceResponse> {
-    throw new Error('method not implemented');
+    const body = await this.bitgo.get(this.url + 'balance').result();
```

Comment:
nit: conventionally, `this.url` would not end with a slash and that would be included in the additional part of the path.
```suggestion
    const body = await this.bitgo.get(this.url + '/balance').result();
```